src/commands.js

```diff
@@ -45,6 +47,13 @@ const startOnly = (appName) => {
   processes.startApp(appName);
 };
 
+const startProfilApps = (profilName) => {
+  const profil = startProfil.getProfilByName(profilName);
+  profil.apps.forEach((appName) => {
```

Comment:
If there is no profile for `profilName` present then `profil` will be undefined and cause an error. Can we add a proper error handling here? (eg. a message `Can't find profile XYZ`)

Reply:
Also this new functionality looks like a good candidate for some tests. Can we add tests for the profile functionality? (in the `/test` folder)